src/uu/expr/src/expr.rs

```diff
@@ -1,6 +1,7 @@
 //* This file is part of the uutils coreutils package.
 //*
 //* (c) Roman Gafiyatullin <r.gafiyatullin@me.com>
+//* (c) Thomas Queiroz <thomasqueirozb@gmail.com>
```

Comment:
we usually don't update the author part for simple changes like this one (but you will be credited in the git history don't worry)

Reply:
Didn't know what was the criteria for that (actually only put that in because I thought it was standard)

Reply:
TBH, I would prefer to move all that list in an AUTHOR file and remove it from the individual files ;)